lib/cli/configuration.js

```diff
@@ -325,3 +326,21 @@ exports.load = (file) => {
     throw error;
   }
 };
+
+/**
+ * @summary Returns defualt configuration file path
+ * @function
+ * @public
+ *
+ * @param {array} fallbacks - file paths for files that may or may not exist
+ * @returns {string} file path
+ *
+ * @example
+ * const configPath = configuration.getDefault([ './foo.js', './bar.js' ]);
+ * console.log(configPath);
+ */
+exports.getDefaultPath = (fallbacks) => {
```

Comment:
Maybe a bit nit-picky, but I'm not sure `getDefaultPath` is such a great name? a) it sounds a bit too generic and b) to me a "path" is usually a directory rather than a file.
This function is basically `getFirstExistingFilename` ;-)  (which I agree is a bit of a mouthful - open to other suggestions)